geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloRunner.scala

```diff
@@ -0,0 +1,35 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0
+  * which accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  *************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo
+
+import org.locationtech.geomesa.tools.accumulo.commands._
+import org.locationtech.geomesa.tools.common.Runner
+import org.locationtech.geomesa.tools.common.commands.{Command, HelpCommand, VersionCommand}
+
+object AccumuloRunner extends Runner {
+  override val scriptName: String = "geomesa-accumulo"
```

Comment:
ScriptName?  Hmm... I wonder if this'll mess with my suggestion 'geomesa' and 'geomesa-accumulo' both work the same...